### ISSUE TITLE
between validation not working

### ISSUE THREAD

**Issue by LeonBeilis** [rakit/validation#112]
I sending value 33 as actionID which has the follow validate rules: required|numeric|between:1,11
edit: also tried min/max also not working

		$validator = new Validator;
		$validation = $validator->make($this->request->data, [
			'planID' => 'required',
			'actionID' => 'required|numeric|between:1,11',
			'value' => 'required|numeric|between:0,1',
		]);
		$validation->validate();
		if ($validation->fails()) {
			Logger::log("updatePrompts", 'data invalid -> ' . json_encode($this->request->data), __FILE__, __LINE__);
			return $this->view->renderFail();
		}
it's still passes the validate.. even if the number is out of boundry..

**Comment by LeonBeilis** [comment]
ok.. my bad found the issue, the value parsed as string.. but still it's weird that it has required numeric and still it got passed, might check this out

**Comment by Sch-Tomi** [comment]
I have the same issue. I think its because the numeric/integer rule doesn't modify the value.

I think it will be usefull to do.

Until then I made a custom rule

```php

use Rakit\Validation\Rule;
use Rakit\Validation\Rules\Interfaces\ModifyValue;

class IntRule extends Rule implements ModifyValue
{
    /** @var string */
    protected $message = "The :attribute must be integer";

    public function modifyValue($value)
    {
        // Do something with $value

        return intval($value);
    }

    /**
     * Check the $value is valid
     *
     * @param mixed $value
     * @return bool
     */
    public function check($value): bool
    {
        return filter_var($value, FILTER_VALIDATE_INT) !== false;
    }
}
```
Then at validation:

```php
    $validator->addValidator('int', new IntRule);

    $validation = $validator->validate($_POST, [
        'year'    => 'required|int|between:1955,2002',
    ]);

```

**Comment by LeonBeilis** [comment]
@Sch-Tomi nice solution, my solution was just cast the param to int.
$a = '1980';
$a = (int) $a;
$validation = $validator->validate($_POST, [
    $a    => 'required|int|between:1955,2002',
]);

**Comment by emsifa** [comment]
Sorry for late reply.

This has been fixes in #119 . Now we can put `numeric` rule to treat numeric string size to its numeric value.
